### DISCUSSION TITLE
Use of Alert group in title

### DISCUSSION THREAD

**Discussion by poxenham-myp** [datalust/seq-tickets#2208] [Help]
Hi there, 

We have many alerts configured for our various applications. I would like to replace those with a single alert and use the group by feature on the Application property instead. One limitation is that there doesn't seem to be a way to use the grouping property in the Alert title or content.

I would like to set the title to  
    High Errors - {Application}

Is this possible?

**Comment by nblumhardt** [comment]
Hi, thanks for dropping us a line.

This isn't generally possible, because each notification may include results for more than one application.

If you're using one of the `Seq.App.Mail.*` apps for the notification, you may be able to index into `Source.Results` to retrieve the group key, e.g. with:

```
High Errors - {[Source.Results[1][1], Source.Results[2][1]], Source.Results[3][1]], if length(Source.Results) > 4 then '...' else undefined()}
```

Untested, as of just now, and may need some tweaks depending on your query, but hopefully a useful starting point to explore from.

Let me know if this helps,
Nick
